version.sh

```diff
@@ -76,3 +76,40 @@ for i in "${!module_names[@]}"; do
 done
 
 echo -e "$res" | column -t -s '|'
+
+echo
+echo "*** Creating requirements file with current emirge module versions"
+
+
+outfile=$(mktemp requirements.txt.XXXXXX)
+
+echo "# requirements.txt created by version.sh" > "$outfile"
+echo "# Date: $(date)" >> "$outfile"
+
+for i in "${!module_names[@]}"; do
```

Comment:
I wonder if there's a way to test this functionality.

Reply:
maybe just run pip install with the created file?

Reply:
Sure, that seems OK. FWIW, if run with `--editable`, is that smart enough to check out the relevant branches in already-existing checkouts? (If not, we should document somewhere that the only way to use this feature is to make a new checkout from scratch.)

Reply:
Yes, it will overwrite the version in an already checked-out repo. Local (uncommitted) changes will still be there after the pip install. Note that what gets recorded in the requirements.txt by version.sh is not a branch, but a commit (or tag).